### PR TITLE
e-Wallet app

### PR DESCRIPTION
**Pull request by GedeonTS**
## e-Wallet app
In this PR I created  a Ruby on Rails application that allows the user to:

- register and log in, so that the data is private to them.
- introduce new transactions associated with a category.
- see the money spent on each category.

Here is the video for descritption and the live demo link: [video](https://www.loom.com/share/2f75bf5ca3364feeb367571d015ec6d5)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,77 @@
...
+# e-Wallet App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div style="display:flex">
+<img src="https://user-images.githubusercontent.com/97834160/189349051-c0d0a462-1e20-4b6d-9677-c9db9aea27c4.png" width="220" height="380">
+<img src="https://user-images.githubusercontent.com/97834160/189348100-e2bd97dd-f38f-41f4-a3ec-ac4fb3be8b9b.png" width="230" height="380">
+<img src="https://user-images.githubusercontent.com/97834160/189348482-8fe3b7b9-b540-4bde-a911-e210c54399be.png" width="230" height="380">
 
-Things you may want to cover:
```
Comment (OLIPLICHE) [inline README.md:8]: - Please Make sure the link to the deployed project is present here in the README. Kindly move it from PR description.

File: app/views/devise/registrations/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
```
Comment (OLIPLICHE) [inline app/views/devise/registrations/edit.html.erb:3]: ## Please make sure the user should be able to register in the app with full name, email and password.
- **In Trying to Register:** 



Capture


- **Outcome:** 
Capture

File: spec/models/expenses_spec.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
```
Comment (OLIPLICHE) [inline spec/models/expenses_spec.rb:2]: ## Please Make sure the Unit and integrated test are all green.
- These bellows are the samples of your test I run locally 
Capture


- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @GedeonTS , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your App runs in Heroku👏 👏
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used 👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (GedeonTS) [reply README.md:8]: Thank you @OLIPLICHE, let me do it right away!

File: spec/models/expenses_spec.rb (hunk 3 above)
Comment (GedeonTS) [reply spec/models/expenses_spec.rb:2]: Screenshot from 2022-09-10 00-13-54

Okay :+1:  let me take another look even though on my side all the tests are runing well.

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @GedeonTS 👋 , 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GedeonTS** [comment]
Thank you @codepantha :+1:

**Pull request merged**